### PR TITLE
Copy guidelines formatting tweaks

### PR DESCRIPTION
**Pull request by leahkoerper**
I made some formatting consistency tweaks to @michelleweber's fine work (from #7588) and added some bold emphasis that I believe was supposed to be there in the original PR. Props also to @karmatosed.

### CODE REVIEW

**Comment by leahkoerper** [comment]
It's been pointed out to me that perhaps using the quote markdown is not technically correct for all the example text. My intention was to delineate them more because I found it hard to parse particularly in the section about bulleted lists. The lists themselves stood out, but the intro text before the list (which was also part of each example) blended right into the guidelines themselves. Is there a better strategy to clarify without using quote markdown?

**Review by tofumatt — APPROVED** [review]
These improvements are solid and I think the quotes make everything more readable, so I'm for them. I'm going to merge this because it's an improvement; sorry it took ages to get to 😓
